src/chipset/intel_i450kx.c

```diff
@@ -324,7 +317,6 @@ mc_write(int func, int addr, uint8_t val, void *priv)
 
     case 0x58:
         dev->mc_pci_conf[addr] = val & 2;
-        mem_set_mem_state_both(0xa0000, 0x20000, (val & 2) ? ENABLE_SEGMENT : DISABLE_SEGMENT);
```

Comment:
Why was this removed?

Reply:
On the MC, that register just enables the A0000-BFFFF segment. Just so I can avoid conflicts with the PB. i450KX works in a way 86Box can't properly support.